### PR TITLE
docs: updated README.md

### PR DESCRIPTION
**Pull request by drugilsberg**
Refactoring README.md to ease reproducibility of the paper results.

### CODE REVIEW

**Comment by drugilsberg** [comment]
hey @jannisborn any new on this?

File: README.md [hunk 1]
```diff
@@ -76,35 +76,68 @@ models
...
 ```sh
 mkdir code && cd code && \
-  git clone --branch sarscov2 https://github.com/PaccMann/paccmann_predictor && \ 
-  git clone --branch 0.0.2 https://github.com/PaccMann/toxsmi && \
-  git clone --branch sarscov2 https://github.com/PaccMann/paccmann_omics && \ 
-  git clone --branch sarscov2 https://github.com/PaccMann/paccmann_chemistry && \ 
   git clone --branch sarscov2 https://github.com/PaccMann/paccmann_generator && \
   cd ..
 ```
-The branch is given to ensure a version working with the provided conda environment.
```
Comment (jannisborn) [inline README.md:94]:
```suggestion
The branch is given to ensure a version working with the provided conda environment.
```

Comment (jannisborn) [reply README.md:94]: I think it makes sense to keep this comment.

**Review by jannisborn — APPROVED** [review]
Thx, looks good, just made one optional suggestion.

**Pull request merged**